example/src/AccountOverviewScreen.tsx

```diff
@@ -23,6 +24,8 @@ const RlyNetwork = RlyMumbaiNetwork;
 
 RlyNetwork.setApiKey(PrivateConfig.RALLY_API_KEY || '');
 
+const customTokenAddress: string | undefined = undefined;
```

Comment:
This is meant to be changed by the developer here in the code (i.e. it's not in the IU or something) right? Maybe worth a comment or a line in the README

Reply:
I'll add a comment here for now. README of this repo isn't really super existent, so doesn't make sense there.